content/blog/2017/2017-04-11-JW-agenda-announce.adoc

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Jenkins World 2017 Agenda is Live"
+tags:
+- event
+- jenkinsworld
+author: alyssat
+---
+We are excited to announce the agenda for link:https://www.cloudbees.com/jenkinsworld/home[Jenkins World 2017]. This year’s event promises to have something for everyone - whether you are a novice, intermediate or advanced user...we’ve got you covered.  
+Jenkins World 2017 consist of 6 tracks, 60+ Jenkins and DevOps sessions, 40+ industry speakers, 16+ training and workshops.
+image:/images/conferences/jenkinsworld-shutterfly-speaking.png[role=center]
+
+Here is a sneak peek of Jenkins World 2017:
+
+=== Show and Tell
+
+- Plugin Development for the Pipeline
+- Extending Blue Ocean
+- How to Use Jenkins Less
```

Comment:
I'm not sure whether this talk title makes sense to list here (at least without a link to it in the agenda, or any further context).